index.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
-module.exports = function (a, b) {
+module.exports = function (a, b, minComp) {
 	if (!Buffer.isBuffer(a) || !Buffer.isBuffer(b)) {
 		throw new TypeError('Arguments must be Buffers');
 	}
 
+	var len = Math.max(a.length, b.length, minComp || 0);
```

Comment:
If I have understood correctly, you are suggesting sth. like this:
```js
var len = minComp || Math.max(a.length, b.length);
```
In that case, `bufferEqualsConstant(Buffer('abc'), Buffer('abd'), 2)` equals to `true` - not what I would expect, because of the `…Equals…`.

Developers which accept this information lost, may do:
```js
bufferEqualsConstant(Buffer('abc').slice(0,2), Buffer('abd').slice(0,2))
// or, depending on the context:
bufferEqualsConstant(Buffer('abc'.slice(0,2)), Buffer('abd'.slice(0,2)))
```